rebar.config

```diff
@@ -13,21 +13,12 @@
     {iso8601, ".*", {git, "https://github.com/erlsci/iso8601.git", {tag, "1.3.1"}}}
 ]}.
 
-{plugins, [rebar3_format, erlfmt]}.
```

Comment:
I like https://github.com/AdRoll/rebar3_format better to manage formater then I apply the whatsapp formater erlfmt

Reply:
Ok, sorry about that. I wasn't sure what the relationship between the two and thought it was just something leftover. I will revert that

Reply:
I reverted locally, but haven't pushed yet because `rebar3_format` doesn't seem provide the equivalent of `erlfmt`'s `--check` flag. With out `--check`, there's no way to check if formatting in CI.